.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "cargo" # See documentation for possible values
```

Comment:
Since we also host all the SDKs in this repo, do you think it would be good to add ecosystem settings for the other languages as well? I know we don't use a many libraries in the SDKs now, but that could change in the future?

Reply:
Sure, I will look into that

Reply:
Alright, `dependabot.yml` has been updated for the sdks languages they support - take a look to make sure it seems right.